### PR TITLE
Update local-development.md

### PR DESCRIPTION
**Pull request by letmejustputthishere**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [x] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [x] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: docs/developer-docs/integrations/bitcoin/local-development.md [hunk 1]
```diff
@@ -105,6 +105,10 @@ The port used can be changed in `dfx.json`.
 :::
 
 - #### Step 2: Deploy the example canister:
+    If you're running the Rust example, use
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/bitcoin/local-development.md:108]:
```suggestion
    If you're running the Rust example, use the command:
```

File: docs/developer-docs/integrations/bitcoin/local-development.md [hunk 2]
```diff
@@ -105,6 +105,10 @@ The port used can be changed in `dfx.json`.
...
+
+        dfx deploy basic_bitcoin --argument '(variant { regtest })'
+    If you're running Motoko, use
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/bitcoin/local-development.md:111]:
```suggestion

    If you're running the Motoko example, use the command:
```

**Review by jessiemongeon1 — APPROVED** [review]

**Review by ielashi — CHANGES_REQUESTED** [review]
We don't need separate commands for the different languages. We just need to fix the Motoko examples's dfx.json. I'll take care of it.

**Comment by letmejustputthishere** [comment]
> We don't need separate commands for the different languages. We just need to fix the Motoko examples's dfx.json. I'll take care of it.

@ielashi the initialisation arguments are still different for the two examples so i don't think there's anything you can do about that in `dfx.json`?

motoko expects [upper case variants](https://github.com/dfinity/examples/blob/d54cd27712450e77e4a398e35f0f0118eee8a47e/motoko/basic_bitcoin/src/basic_bitcoin/basic_bitcoin.did#L9) and rust expects [lower case variants](https://github.com/dfinity/examples/blob/d54cd27712450e77e4a398e35f0f0118eee8a47e/rust/basic_bitcoin/src/basic_bitcoin/basic_bitcoin.did#L9) for the init args.

so imo we still need to merge this PR

**Comment by ielashi** [comment]
> > We don't need separate commands for the different languages. We just need to fix the Motoko examples's dfx.json. I'll take care of it.
> 
> @ielashi the initialisation arguments are still different for the two examples so i don't think there's anything you can do about that in `dfx.json`?
> 
> motoko expects [upper case variants](https://github.com/dfinity/examples/blob/d54cd27712450e77e4a398e35f0f0118eee8a47e/motoko/basic_bitcoin/src/basic_bitcoin/basic_bitcoin.did#L9) and rust expects [lower case variants](https://github.com/dfinity/examples/blob/d54cd27712450e77e4a398e35f0f0118eee8a47e/rust/basic_bitcoin/src/basic_bitcoin/basic_bitcoin.did#L9) for the init args.
> 
> so imo we still need to merge this PR

Both examples should use lower-case invariants, as that what is specified in the interface specification of the Bitcoin API. I addressed this, along with updating dfx.json here: https://github.com/dfinity/examples/pull/606, so we don't the change in this PR.

**Pull request closed without merging**